Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist

```diff
@@ -1521,6 +1582,50 @@ changing “passwordID” to the correct element ID. If the value you typed into
 			<key>pfm_type</key>
 			<string>string</string>
 		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>5.2</string>
+			<key>pfm_description</key>
+			<string>In some Active Directory environments, users do not use username@domain to login; they use a UPN suffix to make the username easier to use. This setting maps the UPN suffix to the correct AD domain name. For example, if you have an AD domain of foo.com but want users to sign in as user@bar.com, a UPN suffix of foo.com is created in AD and the user account is set to user@foo.com. This setting then would map foo.com to bar.com by setting the key upn to foo.com and the domain key to bar.com.</string>
+			<key>pfm_name</key>
+			<string>upnSuffixToDomainMappings</string>
+			<key>pfm_subkeys</key>
+			<array>
+				<dict>
+					<key>pfm_hidden</key>
```

Comment:
I'm not sure this provides any benefit here - does it make any difference for you? (I'm aware this is a ProfileCreator-only key).